app/Department.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace App;
+
+use Illuminate\Database\Eloquent\Model;
+
+class Department extends Model
+{
+	public function users() {
+		return $this->hasMany('App\User');
+	}
+
+	public function organisation() {
```

Comment:
Change to 'organisations' because an HR department can exist in Oracle and HP for example.